A14-channelz.md

```diff
@@ -728,8 +736,49 @@ message ChannelData {
   google.protobuf.Timestamp last_call_started_timestamp = 7;
 }
 
+// A trace event is an interesting thing that happened to a channel or
+// subchannel, such as creation, address resolution, subchannel creation, etc.
+message ChannelTraceEvent {
```

Comment:
I suggest leaving these new message types out of this doc, since they weren't really part of the original channelz proposal.  Instead, let's just leave an empty `ChannelTrace` message with a comment that the details will be filled in by the channel tracing proposal (with a link to the gRFC).

Reply:
Done